src/main/java/Casper.java

```diff
@@ -0,0 +1,130 @@
+    private static final Task[] taskList = new Task[100];
+    private static int noOfTasks = 0;
+    private static void wrapEchoMessage(String message){
+        System.out.println(separator);
+        System.out.println("     "+message);
+        System.out.println(separator);
+    }
+    private static void echoGreetings(){
+        String logo = "       ___ __ _ ___ _ __   ___ _ __\n"
+                + "      / __/ _` / __| '_ \\ / _ \\ '__|\n"
+                + "     | (_| (_| \\__ \\ |_) |  __/ |   \n"
+                + "      \\___\\__,_|___/ .__/ \\___|_|   \n"
+                + "                   | |              \n"
+                + "                   |_|               ";
+        System.out.println("    Starting\n" + logo);
+        wrapEchoMessage("Boo! I'm Casper!\n     What can I do for you?");
+    }
+    private static void handleQueries(){
+        Scanner inputScanner = new Scanner(System.in);
+        while (true) {
+            String userInput = inputScanner.nextLine();
+            if (userInput.equals("bye")) {
+                break;
+            }
+            if (userInput.equals("list")) {
+                echoTaskList();
+            } else if(userInput.startsWith("mark")) {
+                int targetTaskNumber = validateMarkInput(userInput);
+                if (targetTaskNumber != -1) {
+                    taskList[targetTaskNumber-1].markTask();
+                }
+            } else if (userInput.startsWith("unmark")) {
+                int targetTaskNumber = validateMarkInput(userInput);
+                if (targetTaskNumber != -1) {
+                    taskList[targetTaskNumber-1].unMarkTask();
+                }
+            } else if (userInput.startsWith("event")){
+                Task newTask = getEvent(userInput);
+                taskList[noOfTasks] = newTask;
+                noOfTasks++;
+                wrapEchoMessage("Got it. I've added this task: \n       "
+                        +newTask+"\n     Now you have "+noOfTasks+" tasks in the list");
+            }else if(userInput.startsWith("deadline")){
+                Task newTask = getDeadline(userInput);
+                taskList[noOfTasks] = newTask;
+                noOfTasks++;
+                wrapEchoMessage("Got it. I've added this task: \n       "
+                        +newTask+"\n     Now you have "+noOfTasks+" tasks in the list");
+            }else if(userInput.startsWith("todo")){
+                Task newTask = getTodo(userInput);
+                taskList[noOfTasks] = newTask;
+                noOfTasks++;
+                wrapEchoMessage("Got it. I've added this task: \n       "
+                        +newTask+"\n     Now you have "+noOfTasks+" tasks in the list");
+            }else{
+                wrapEchoMessage("Pardon? I didn't get your message.");
+            }
+        }
+    }
+
+    private static Task getEvent(String userInput) {
```

Comment:
Good job on the indentation for the various lines of code but you can consider reducing arrowhead nature